specs/agents/mobile/events.md

```diff
@@ -0,0 +1,53 @@
+## Mobile Events
+
+This document describes event used by the Mobile SDKs using the [OpenTelementry Event Api](https://github.com/open-telemetry/opentelemetry-specification/blob/0a4c6656d1ac1261cfe426b964fd63b1c302877d/specification/logs/event-api.md).
+Event names will be recording using the `event.name`
+
+
+### `ApplicationLaunch` 
+This event represents the occurrence of app entering the foreground or background or other application lifecycle states.
+
+#### Attributes
+| Name         | Type   | Values                 | Description                                        |
+|--------------|--------|------------------------|----------------------------------------------------|
+| `event.name` | String | `ApplicationLifecycle` | The name of the event                              |
+| `state`      | String | `foreground`           | The lifecycle state the app is transitioning to.   | 
+| `state.last` | String | `background`           | The lifecycle state the app is transitioning from. | 
+
+
+
+### Application Non-Responsive (ANR)
+This event represents the occurrence of an 'ANR' error reported by the Android OS.
+
+#### Attributes
+| Name           | Type | Value | Description |
+|----------------|------|-------|-------------|
+| `event.name`   |      |       |             |
+
+### `Breadcrumbs`
+
+An event that allows customers to add events to a common `Event` that allows them to retrace users' steps. 
+
+#### Attributes
+| Name           | Type    | Value        | Description            |
+|----------------|---------|--------------|------------------------|
+| `event.name`   | String  | `Breadcrumb` | The name of the Event  |
+
+### Crashes
+
+This event represent a crash event
+
+#### Attributes
+| Name           | Type    | Values  | Description    |
+|----------------|---------|---------|----------------|
+| `event.name`   | String  | `Crash` | The event name | 
+| `stacktrace`   | String  | N/A     | A Stacktrace   | 
+
+### `Application Opens`
+
+An event that represents when an app is opened.
+
+#### Attributes
+| Name           | Type   | Value                | Description    |
+|----------------|--------|----------------------|----------------|
+| `event.name`   | String | `Application Opened` | The event name | 
```

Comment:
```suggestion
| `event.name`   | String | `application_opened` | The event name | 
```

Reply:
+1 - I also think we should have an extra parameter along with this event to denote whether the app's been opened from scratch or just re-opened. Maybe something like `mode: (started/resumed)`?

Reply:
Yeah that's true, so you think we should also send the `state`s defined above as part of this event as well? Or you're talking about making a query that brings this event's occurrences based off of the latest state received from the device? The latter sounds a bit complicated

Reply:
yeah, I think it would be pretty hard to determine the state transitions purely based on which lifecycle event happened before... it could have occurred days ago depending on the app usage. It makes more sense to have an attribute on the event noting it

Reply:
I removed this specific event category in favor of application lifecycle as we discussed this morning.